tests/graph/test_graph_basic.py

```diff
@@ -0,0 +1,38 @@
+from mim_ocr.backends.google_vision import GCPBackend
```

Comment:
W przykładzie jest użyty TesseractBackend (i zdecydowanie wolę, żeby tak zostało). Czy jest jakiś powód, dla którego tutaj używamy GCP?

Reply:
Czysto z przyzwyczajenia, bo GCP generalnie działa dużo lepiej. Zmieniłem w każdym razie na Tesseract, bo w tym teście to nie ma znaczenia większego. (Jak dobrze rozumiem, to GCP za każde użycie ściąga coś z portfela, więc do pipeline'u lepiej wrzucić Tesseract?)

Reply:
Dokładnie tak.